test/list.go

```diff
@@ -486,6 +488,75 @@ func DescribeListTestsFor(ctx *common.TestContext, t TestCase, responseMode Resp
 				})
 			}
 
+			if t.SupportsAsyncOperations {
+				Context("when attach_last_operations is truthy", func() {
+					It("retrieves the resources list when each resource contains the last operation it is associated to", func() {
+						resp := ctx.SMWithOAuth.GET(t.API).
+							WithQuery("attach_last_operations", "true").
+							Expect().Status(http.StatusOK).
+							JSON()
+						for _, resource := range resp.Path("$.items[*]").Array().Iter() {
+							currentResourceId := resource.Object().Value("id").String().Raw()
+							currentResourceState := resource.Object().Value("ready").Boolean().Raw()
+							lastOp := resource.Object().Value("last_operation")
+
+							Expect(lastOp.Object().Value("resource_id").String().Raw()).To(Equal(currentResourceId))
+							Expect(lastOp.Object().Value("ready").Boolean().Raw()).To(Equal(currentResourceState))
```

Comment:
what is the point of comparing the "ready" field?

Reply:
and how do we know that this is really the last operation on that resource and not just one of them (not the last)?